lib/rules/multiline-html-element-content-newline.js

```diff
@@ -108,6 +114,12 @@ module.exports = {
         }
 
         const getTokenOption = { includeComments: true, filter: (token) => token.type !== 'HTMLWhitespace' }
+        if (ignoreWhenEmpty && node.children.length === 0) {
```

Comment:
What do you think about using single `if` statement?
```js
if (
  ignoreWhenEmpty &&
  node.children.length === 0 &&
  template.getFirstTokensBetween(node.startTag, node.endTag, getTokenOption).length === 0
) {
  return
}

Reply:
i wanted to make it clean but with this wrapping its looks ok 👍